### DISCUSSION TITLE
Unpause DAG without creating a DAG run for past schedule

### DISCUSSION THREAD

**Discussion by kmehkeri** [apache/airflow#35736] [Q&A]
It seems that depending on `catchup` unpausing a DAG that has a schedule will create at least one (if `catchup=False`) or multiple (if `catchup=True`) DAG runs for past schedules.

I would like to unpause a DAG without creating any runs. If I want it to execute at midnight and unpause it now, I would really like it to run next midnight, not right now for the past one... Is this possible?

**Comment by edegaudenzi** [comment]
Related to https://github.com/apache/airflow/pull/8776
